config/README.md

```diff
@@ -2,11 +2,23 @@
 
 ## Labels
 
-[labels.json](./labels.json) list common default labels for repositories in the blindnet-io organization.
-We use this file with [github-label-sync](https://github.com/Financial-Times/github-label-sync) to update labels in existing repositories whenever necessary.
+We use [github-label-sync](https://github.com/Financial-Times/github-label-sync) to update labels in existing repositories whenever necessary.
+
+Common default labels for repositories in the blindnet-io organization are configured in:
+
+- [global-labels.json](./global-labels.json) for all repositories
+- [com-labels.json](./com-labels.json) for communication repositories
+
+To update and add labels on a specific repository, run:
 
 ```bash
 npm i -g github-label-sync
 
-github-label-sync --allow-added-labels --labels global-labels.json --access-token xxxxx blindnet-io/REPO
+github-label-sync --allow-added-labels --labels global-labels.json --access-token "${GHKEY}" blindnet-io/REPO
```

Comment:
`GHKEY` is personal access token right?

Reply:
Yep exactly. I didn't take time to thoroughly document this as it's just a temporal solution, mostly for myself to deal with (see https://github.com/blindnet-io/devrel-management/issues/12#issuecomment-1145419858). Should I add a sentence to explain this? (with a link to the doc etc.)

Reply:
Added this to the README with a new script.